source/community/release_notes/palm.rst

```diff
@@ -15,6 +15,46 @@ These are the release notes for the Palm release, the 16th community release of
 Breaking Changes
 ****************
 
+Tutor minimum versions of Docker and Compose
+============================================
+
+In Palm the minimum required versions will be `Docker v20.10.15 <https://docs.docker.com/engine/release-notes/20.10/#201015>`_ and Compose v2.0.0. These versions date back from May 2022, so not everyone might be able to upgrade. We'll do our best to keep backward compatibility with non-buildkit installations, but users will not have access to the fanciest features, such as build-time cache.
+
+RateXBlock unsupported
+======================
+
+The RateXBlock has been deprecated and is now officially unsupported. `FeedbackXBlock 
+<https://github.com/pmitros/FeedbackXBlock>`_ is the recommended replacement.
```

Comment:
Is this an officially supported block? This isn't in the Open edX org

Reply:
The repo was moved to openedx-unsupported as part of the deprecation process: https://github.com/openedx-unsupported/RateXBlock